content/docs/rendering-elements.md

```diff
@@ -1,75 +1,82 @@
+Nadat je een element hebt gemaakt, kun je zijn children of attributen niet meer wijzigen. 
+Een element is als een enkel frame in een film: het vertegenwoordigt de UI op een bepaald moment in de tijd.
 
-With our knowledge so far, the only way to update the UI is to create a new element, and pass it to `ReactDOM.render()`.
+Voor zover we nu weten, kan de UI alleen bijgewerkt worden door een nieuwe element te maken en het door te geven aan `ReactDOM.render()`.
 
-Consider this ticking clock example:
+Neem dit voorbeeld van een tikkende klok:
 
 `embed:rendering-elements/update-rendered-element.js`
 
-[](codepen://rendering-elements/update-rendered-element)
+[Probeer het op CodePen](codepen://rendering-elements/update-rendered-element)
 
-It calls `ReactDOM.render()` every second from a [`setInterval()`](https://developer.mozilla.org/en-US/docs/Web/API/WindowTimers/setInterval) callback.
+Hierin wordt `ReactDOM.render()` elke seconde aangeroepen via een [`setInterval()` (Engels)](https://developer.mozilla.org/en-US/docs/Web/API/WindowTimers/setInterval) callback.
 
->**Note:**
+>**Opmerking:**
 >
->In practice, most React apps only call `ReactDOM.render()` once. In the next sections we will learn how such code gets encapsulated into [stateful components](/docs/state-and-lifecycle.html).
+>In de praktijk roepen de meeste React apps `ReactDOM.render()` maar één keer aan. 
+In de volgende hoofdstukken zullen we leren hoe code zoals dit voorbeeld ingekapseld worden in [stateful componenten](/docs/state-and-lifecycle.html).
 >
->We recommend that you don't skip topics because they build on each other.
+>We raden aan om geen hoofdstukken over te slaan, omdat ze op elkaar voortbouwen.
 
-## React Only Updates What's Necessary {#react-only-updates-whats-necessary}
+## React Werkt Alleen Datgene Bij Dat Nodig Is {#react-only-updates-whats-necessary}
 
-React DOM compares the element and its children to the previous one, and only applies the DOM updates necessary to bring the DOM to the desired state.
+Het React DOM vergelijkt het element en zijn children met het vorige element en voert alleen de wijzigingen in het DOM uit die nodig zijn om deze in de gewenste staat te krijgen.
 
-You can verify by inspecting the [last example](codepen://rendering-elements/update-rendered-element) with the browser tools:
+Je kunt dit zien door met de browser tools het [laatste voorbeeld](codepen://rendering-elements/update-rendered-element)  te inspecteren:
 
-![DOM inspector showing granular updates](../images/docs/granular-dom-updates.gif)
+![DOM inspector toont granulaire updates](../images/docs/granular-dom-updates.gif)
 
-Even though we create an element describing the whole UI tree on every tick, only the text node whose contents has changed gets updated by React DOM.
+Ook al maken we een element dat de hele UI tree beschrijft op elke tick, alleen de tekst node waarvan de inhoud wijzigt, wordt bijgewerkt door het React DOM.
```

Comment:
```suggestion
Ook al maken we een element dat de hele UI tree beschrijft op elke tick, wordt alleen de tekst node waarvan de inhoud wijzigt, bijgewerkt door het React DOM.
```

Reply:
I agree that my translation isn't very clear, but the suggestion doesn't sound "correct" to me.

Reply:
Alternate suggestion:
```Alleen de tekst node waarvan de inhoud wijzigt, wordt bijgewerkt door het React DOM, ook al gebruiken we een element dat de hele UI tree beschrijft op elke tick.```

I think that in Dutch it doesn't always work to put the subordinate clause in front of the main clause. By switching them around it feels more natural to me, even though it slightly changes the intention of the English sentence.

What do you think, @jochenboesmans?

Reply:
Remove the first comma and it should be good 👍